ts/Extensions/Annotations/Popup.ts

```diff
@@ -1148,23 +1482,30 @@ H.Popup.prototype = {
         },
         /**
          * Extract full name and type of requested indicator.
+         *
          * @private
+         *
          * @param {Highcharts.Series} series
-         * Series which name is needed. (EDIT mode - defaultOptions.series, ADD
-         * mode - indicator series).
-         * @param {string} - indicator type like: sma, ema, etc.
-         * @return {Object} - series name and type like: sma, ema, etc.
+         *        Series which name is needed(EDITmode - defaultOptions.series,
+         *        ADDmode - indicator series).
+         *
+         * @param {string} [IndicatorType]
+         *        Type of the indicator i.e. sma, ema...
+         *
+         * @return {Highcharts.IndicatorNameCouple}
+         *        Full name and series type.
```

Comment:
Type `Highcharts.Dictionary<string>` is enough, no need to introduce an extra type for generics.